config/engines.yml

```diff
@@ -191,6 +191,16 @@ radon:
     - \.py$
   default_ratings_paths:
     - "**.py"
+reek:
+  image: codeclimate/codeclimate-reek
+  description: "Reek examines Ruby classes, modules, and methods and reports any code smells it finds."
+  community: true
+  upgrade_languages:
+    - Ruby
```

Comment:
We've tended to do it on other engines just because it makes it an easier change if we decide we do want to enable them by default, I think. The same question would apply to the `enable_regexps` field, and we add that for every engine.